.scripts/ci.yml

```diff
@@ -10,13 +10,13 @@ parameters:
     default: true
   - name: OSVmImage
     type: string
-    default: ubuntu-18.04
+    default: ubuntu-20.04
```

Comment:
why are we upgrading the image versions ?

Reply:
There would be 1) one [error ](https://dev.azure.com/azure-sdk/public/_build/results?buildId=1837591&view=logs&j=39c431fc-b8dd-5dce-1f4e-2e5be8576b0f)happened if we didn't upgrade macOS image versions. Also 2) one [warning ](https://dev.azure.com/azure-sdk/public/_build/results?buildId=1837647&view=logs&j=6e6cd934-b249-5120-4299-21f24be20cc5)to upgrade ubuntu versions.

So I resolved them by upgrading both systems